apps/desktop-wallet/package.json

```diff
@@ -69,18 +68,10 @@
     "concurrently": "^7.4.0",
     "cross-env": "^7.0.3",
     "dayjs": "^1.10.7",
-    "electron": "^22.3.5",
+    "electron": "22.3.5",
```

Comment:
I had an issue where the solution was to remove the caret, but I can't find the link anymore...